DESCRIPTION

```diff
@@ -31,8 +31,11 @@ Imports:
     digest,
     crosstalk,
     reactR,
-    purrr
+    purrr,
+    withr (>= 2.4.2)
 URL:
     https://rstudio.github.io/rscpages,
     https://github.com/rstudio/rscpages
 VignetteBuilder: knitr
+Remotes:  
```

Comment:
I don't think the DESCRIPTION can contain this [when it goes to CRAN](https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html#cran-submission)--do we need something in particular from the dev version?